processing/src/main/java/org/apache/druid/segment/nested/NestedCommonFormatColumn.java

```diff
@@ -17,27 +17,31 @@
  * under the License.
  */
 
-package org.apache.druid.segment.column;
+package org.apache.druid.segment.nested;
```

Comment:
Package nit: I wonder if it shouldn't be `segment.column.nested`?  Probably doesn't really make that much of a difference, but seems more technically accurate.

Reply:
we don't have a `segment.column.nested` right now, i almost left it in `segment.column` since that also seems appropriate